bin/ember-template-lint.js

```diff
@@ -354,10 +369,15 @@ async function run() {
   if (options.printPending) {
     return printPending(results, options);
   } else {
-    if (results.errorCount || results.warningCount || (options.includeTodo && results.todoCount)) {
+    if (
+      results.errorCount ||
+      results.warningCount ||
+      (options.includeTodo && results.todoCount) ||
+      (options.updateTodo && todoInfo.added)
+    ) {
```

Comment:
This is getting a tad complicated to reason about. Maybe break this down into some local variables?

Something like:

```js
let hasErrors = results.errorCount > 0;
let hasWarnings = results.warningCount > 0;
let hasTodoInfo = options.includeTodo && results.todoCount > 0;
let hasUpdatedTodos = options.updateTodo && todoInfo.added > 0;

if (hasErrors || hasWarnings || hasTodoInfo || hasUpdatedTodos) {
```

FWIW, I think this may also allow you to remove the `.eslintrc.js` change...

Reply:
Oh, also, I think we also need to track removals that are **done** by `--fix` so there is an additional case to account for here.

Reply:
> FWIW, I think this may also allow you to remove the .eslintrc.js change...

It does not, as that was triggered in `pretty.js` too, unfortunately.

Reply:
> Oh, also, I think we also need to track removals that are done by --fix so there is an additional case to account for here.

`--update-todo` and `--fix` are mutually exclusive, so I don't think that's the case. I do think we should error when these flags are used in conjunction.

Reply:
Sure, that seems fine, but that wasn't what I meant.

I'm saying we output in stdout whenever we mutate todos, which means `--fix` will also need to be able to trigger some content here (e.g. `2 todos removed`).

Reply:
Oh sorry I did misinterpret what you were saying. We can additionally track removals for `--fix`.

Reply:
See my comment [here](https://github.com/ember-template-lint/ember-template-lint/pull/1688#discussion_r556815114) about supporting `--fix` and LMKWYT.